### PR TITLE
Paginate and add cell background color for table visualization

### PR DESCRIPTION
**Pull request by jennygaoo**
TODO: in another PR, I'll change the cost difference column so that it'll be titled "amount saved using postal injection" and have the cost differences where postal injection is cheaper show up as positive values. 

Also, I will try to move the negative sign outside of the dollar sign for the cost difference and postal injection columns. 

Also: right now, there are 10 orders displayed per page. I'm thinking about adding a section where the user can select how many orders they want to see per page. 
image

### CODE REVIEW

File: postal-injection/src/main/webapp/summary_table.js [hunk 1]
```diff
@@ -78,18 +78,42 @@ function drawZipCodeSummaryTable(summaryTableData) {
...
 
   // draws table
   const table =
       new google.visualization.Table(document.getElementById('data_table'));
-  table.draw(
-      data,
-      {allowHtml: true, showRowNumber: true, width: '100%', height: '100%'});
+  table.draw(data, {
+    allowHtml: true,
+    pageSize: 10,
```
Comment (jgao23) [inline postal-injection/src/main/webapp/summary_table.js:112]: define pageSize as a static class constant, along with showRowNumber width and height

File: postal-injection/src/main/webapp/summary_table.js [hunk 2]
```diff
@@ -78,18 +78,42 @@ function drawZipCodeSummaryTable(summaryTableData) {
...
+  cellBackgroundColorFormatter.format(data, 2)
 
   // draws table
   const table =
       new google.visualization.Table(document.getElementById('data_table'));
-  table.draw(
-      data,
-      {allowHtml: true, showRowNumber: true, width: '100%', height: '100%'});
+  table.draw(data, {
+    allowHtml: true,
```
Comment (jgao23) [inline postal-injection/src/main/webapp/summary_table.js:111]: why do you need to allow html?

File: postal-injection/src/main/webapp/summary_table.js [hunk 3]
```diff
@@ -78,18 +78,42 @@ function drawZipCodeSummaryTable(summaryTableData) {
...
+
+  /* Coloring the cost difference cell red if postal injection costs less than
+  regular, coloring it yellow if both are equal, and red if postal injection
+  costs more than regular */
+  const cellBackgroundColorFormatter = new google.visualization.ColorFormat();
+  /* null= +/- infinity depending on whether it's used for an upper or lower
+  bound */
+  cellBackgroundColorFormatter.addRange(
+      /* lower bound= */ null, /* upper bound= */ 0,
+      /* text color= */ 'black', /* light green background color= */ '#eeffe9');
```
Comment (jgao23) [inline postal-injection/src/main/webapp/summary_table.js:96]: define the colors like `#eeffe9` as class constants as well as text color.

File: postal-injection/src/main/webapp/summary_table.js [hunk 4]
```diff
@@ -78,18 +78,42 @@ function drawZipCodeSummaryTable(summaryTableData) {
     }
   }
 
-  /* Formatting table. Can't apply number formatting to current cost difference
-  & postal injection columns because they could be "N/A" */
-  const formatter = new google.visualization.NumberFormat({prefix: '$'});
+  /* Adding a dollar sign in front of prices. Can't apply this to current cost
```
Comment (jgao23) [inline postal-injection/src/main/webapp/summary_table.js:81]: this function is getting long. might be more readable if you split up the sections into helper functions if possible.

File: postal-injection/src/main/webapp/summary_table.js (hunk 2 above)
Comment (jennygaoo) [reply postal-injection/src/main/webapp/summary_table.js:111]: it helps the "$" sign display

File: postal-injection/src/main/webapp/summary_table.js [hunk 5]
```diff
@@ -78,18 +89,43 @@ function drawZipCodeSummaryTable(summaryTableData) {
...
+      /* text color= */ BLACK, /* background color= */ LIGHT_RED);
+  cellBackgroundColorFormatter.addRange(
+      /* lower bound= */ 0, /* upper bound= */ 0,
+      /* text color= */ BLACK, /* background color= */ LIGHT_YELLOW);
+  cellBackgroundColorFormatter.addRange(
+      /* lower bound= */ 0.01, /* upper bound= */ null,
+      /* text color= */ BLACK, /* background color= */ LIGHT_GREEN);
+
+  // apply formatter to the cost difference column
+  cellBackgroundColorFormatter.format(data, 2)
```
Comment (andreahu12) [inline postal-injection/src/main/webapp/summary_table.js:116]: I think you're missing a semicolon here. Add parameter comment for the 2?

File: postal-injection/src/main/webapp/summary_table.js [hunk 6]
```diff
@@ -78,18 +89,43 @@ function drawZipCodeSummaryTable(summaryTableData) {
...
-  /* Formatting table. Can't apply number formatting to current cost difference
-  & postal injection columns because they could be "N/A" */
-  const formatter = new google.visualization.NumberFormat({prefix: '$'});
+  /* Adding a dollar sign in front of prices. Can't apply this to current cost
+  difference & postal injection columns because they could be "N/A" */
+  const dollarSignFormatter =
+      new google.visualization.NumberFormat({prefix: '$'});
   // apply formatter to regular shipping cost column
-  formatter.format(data, 3);
+  dollarSignFormatter.format(data, 3);
```
Comment (andreahu12) [inline postal-injection/src/main/webapp/summary_table.js:97]: add parameter comment for the 3?

File: postal-injection/src/main/webapp/summary_table.js (hunk 4 above)
Comment (jennygaoo) [reply postal-injection/src/main/webapp/summary_table.js:92]: will do this in another PR

File: postal-injection/src/main/webapp/summary_table.js [hunk 7]
```diff
@@ -3,6 +3,16 @@
...
 
+const ALLOW_HTML = true;
+const BLACK = 'black';
+const HEIGHT = '100%';
+const LIGHT_RED = '#eeffe9';
+const LIGHT_YELLOW = '#fbffc1';
+const LIGHT_GREEN = '#ffe8e7';
+const NUMBER_OF_ORDERS_PER_PAGE = 10;
+const SHOW_ROW_NUMBER = true;
+const WIDTH = '100%';
```
Comment (jgao23) [inline postal-injection/src/main/webapp/summary_table.js:14]: appreciate the a/b ordering :')

**Pull request merged**